packages/hash/api/src/model/ontology/data-type.model.ts

```diff
@@ -58,8 +78,9 @@ export default class {
    *   Depends on the RFC captured by:
    *   https://app.asana.com/0/1200211978612931/1202464168422955/f
    *
-   * @param params.ownedById - the id of the owner of the entity type
+   * @param params.ownedById - the id of the owner of the data type
```

Comment:
```suggestion
   * @param params.ownedById - the id of the account that owns the data type
```

Reply:
Addressed in https://github.com/hashintel/hash/pull/1197/commits/51c3a539be68ca7bf1c7ffa8f13550e6f20efee4